### PR TITLE
[#367] Sign up entity/stakeholder adaptations

### PR DESCRIPTION
**Pull request by tangrammer**
# Stakeholder steps:
Screenshot 2021-09-07 at 15 17 55
Screenshot 2021-09-07 at 15 18 02
Screenshot 2021-09-07 at 15 18 06

# entity steps
Screenshot 2021-09-07 at 15 19 23
Screenshot 2021-09-07 at 15 19 30
Screenshot 2021-09-07 at 15 19 35
Screenshot 2021-09-07 at 15 19 39
Screenshot 2021-09-07 at 15 19 43

### CODE REVIEW

File: frontend/src/modules/signup/stakeholderSchema.js [hunk 1]
```diff
@@ -118,35 +79,15 @@ export const schema = {
...
-        "orgDescription",
-        "orgUrl",
-        "orgLogo",
-      ],
+      required: ["orgName"],
       properties: {
         orgName: {
-          title: "What is the name of your organisation?",
+          title: "Search Your Entity",
           type: "string",
```
Comment (wayangalihpratama) [inline frontend/src/modules/signup/stakeholderSchema.js:86]: This entity field will be a selected dropdown or a string field?

File: frontend/src/modules/signup/stakeholderSchema.js [hunk 2]
```diff
@@ -118,35 +79,15 @@ export const schema = {
...
         },
-        orgRepresentative: {
-          title: "Which representative groups fits your organisation?",
+        memberName: {
+          title: "Is your Entity not a GPML Member yet?",
           type: "string",
```
Comment (wayangalihpratama) [inline frontend/src/modules/signup/stakeholderSchema.js:90]: I'm not sure, this field will be a select dropdown or string

**Comment by wayangalihpratama** [comment]
I left view comments on this just to ensure if that field will be updated on the next PR or include in this PR. And I also try to submit the form (both entity/stakeholder) return error on javascript (ignore this if this part not including in this PR).

Other than that it looks great!

File: frontend/src/modules/signup/stakeholderSchema.js (hunk 1 above)
Comment (tangrammer) [reply frontend/src/modules/signup/stakeholderSchema.js:86]: WIP for the next PR

File: frontend/src/modules/signup/stakeholderSchema.js (hunk 2 above)
Comment (tangrammer) [reply frontend/src/modules/signup/stakeholderSchema.js:90]: WIP for the next PR

**Comment by tangrammer** [comment]
thanks @wayangalihpratama !

**Pull request merged**